hack/jenkins/verify.sh

```diff
@@ -32,6 +32,5 @@ export PATH=${GOPATH}/bin:${HOME}/third_party/etcd:/usr/local/go/bin:$PATH
 
 # Install a few things needed by the verification tests.
 command -v etcd &>/dev/null || ./hack/install-etcd.sh
-go get -u github.com/tools/godep
```

Comment:
expected some more calls to `kube::util::ensure_godep_version` in the changed files above. All of them have that already but were overriding the ensure godep version?

Reply:
All of these `hack/jenkins/` scripts are trying to be smart and pre-load godep for you. However, on first use, we will run the ensure function to install it. We don't need it pre-loaded. The only script that didn't do this properly was `verify-godeps.sh` and I've added it there. Better to have the ensure function as close to the using script.

Reply:
Alright. That's what I was hoping.